src/library/scala/collection/IndexedSeqView.scala

```diff
@@ -40,7 +40,7 @@ object IndexedSeqView {
   }
 
   /** An `IndexedSeqOps` whose collection type and collection type constructor are unknown */
-  type SomeIndexedSeqOps[A] = IndexedSeqOps[A, AnyConstr, _]
+  type SomeIndexedSeqOps[A] = IndexedSeqOps[A, Any, _]
```

Comment:
That won't compile with Dotty because `Any` is not poly-kinded there.

Reply:
Well, yes, we'll need to change the sources for Dotty. We have to do that anyway, because the definition of `AnyConstr` wouldn't compile, either. What's the current replacement anyway?

Reply:
I was actually thinking of the original definition `type AnyConstr = Any` but now we have `type AnyConstr[X] = Any` which would cross-compile.

But we still shouldn't have `scala.collection.AnyConstr`. This is an implementation detail that is not supposed to leak into user code and it is not specific to collections. If we want to keep this abstraction it either needs to be properly supported (e.g. adding `scala.AnyConstr` in 2.13 and Dotty) or hidden away (is it possible?)

Reply:
Just write a type lambda, since Scala 2 doesn't have nice syntax for them you need to use the projection trick `({ type L[A] = Any })#L`

Reply:
Using a private type alias would probably work too.

Reply:
Looks promising. I've opened https://github.com/scala/scala/pull/7077 to make it private. We can still consider better solutions for RC1.